### PR TITLE
Feature/31 guidesite page

### PR DESCRIPTION
**Pull request by seanchvz**
### Summary
- Finished the implementation according to the provided UI design.
- The layout and components now match the design specifications.

### CODE REVIEW

**Comment by shunindesuuu** [comment]
Looks good to me, how about you @miggyalino ?

File: src/app/guidesite/page.tsx [hunk 1]
```diff
@@ -0,0 +1,363 @@
...
+    guidesiteData.requestDocuments;
+
+  const mainTextWithLink = mainText.replace(
+    'link',
+    `<a href="${linkUrl}" class="text-blue-500 underline" target="_blank" rel="noopener noreferrer">${linkText}</a>`
+  );
+
+  return (
+    <div>
+      <h2 className="text-bold font-bold mb-4 pt-10">{title}</h2>
```
Comment (miggyalino) [inline src/app/guidesite/page.tsx:166]: Put this in a separate file since this is a separate component

File: src/app/guidesite/page.tsx [hunk 2]
```diff
@@ -0,0 +1,363 @@
...
+    const numeral = key as keyof typeof numeralLookup; // Cast key for TypeScript
+    while (num >= numeralLookup[numeral]) {
+      roman += romanSymbols[numeral]; // Use the symbol from the romanSymbols object
+      num -= numeralLookup[numeral];
+    }
+  }
+  return roman;
+};
+
+const AssetsFacilitiesTable = () => (
```
Comment (miggyalino) [inline src/app/guidesite/page.tsx:39]: Pwede lang ito 
`const AssetsFacilitiesTable = (.......)`

File: src/app/guidesite/page.tsx [hunk 3]
```diff
@@ -0,0 +1,363 @@
...
+              </ol>
+            )}
+          </td>
+        </tr>
+      ))}
+    </tbody>
+  </table>
+);
+
+const LibraryFacilities = () => (
```
Comment (miggyalino) [inline src/app/guidesite/page.tsx:65]: Same here

File: src/app/guidesite/page.tsx [hunk 4]
```diff
@@ -0,0 +1,363 @@
...
+          {facility.nestedInstructions.map((instruction, idx) => (
+            <li key={idx}>{instruction}</li>
+          ))}
+        </ol>
+      </li>
+    ))}
+  </ol>
+);
+
+const SuzanDoromalFacilities = () => (
```
Comment (miggyalino) [inline src/app/guidesite/page.tsx:98]: Same here

File: src/app/guidesite/page.tsx [hunk 5]
```diff
@@ -0,0 +1,363 @@
...
+        {content.map((equipment, index) => (
+          <li key={index} style={{ fontWeight: 'bold' }}>
+            <span style={{ fontWeight: 'normal' }}>{equipment.item}</span>
+            {equipment.details && (
+              <ol className="list-outside list-[lower-alpha] ml-14">
+                <li style={{ fontWeight: 'bold' }}>
+                  <span style={{ fontWeight: 'normal' }}>
+                    {equipment.details}
+                  </span>
+                </li>
```
Comment (miggyalino) [inline src/app/guidesite/page.tsx:142]: Why not use tailwind for font weights? https://tailwindcss.com/docs/font-weight

File: src/app/guidesite/page.tsx [hunk 6]
```diff
@@ -0,0 +1,363 @@
...
+            <span className="font-bold">{document.item}</span>
+            {document.details && (
+              <>
+                {' '}
+                -- <span className="font-normal">{document.details}</span>
+              </>
+            )}
+          </li>
+        ))}
+      </ul>
```
Comment (miggyalino) [inline src/app/guidesite/page.tsx:189]: Use tailwind for font weight

File: src/app/guidesite/page.tsx [hunk 7]
```diff
@@ -0,0 +1,363 @@
...
+      <div className="mt-8">
+        <ul className="list-decimal list-outside ml-10">
+          <span className="font-bold block">
+            Office of the Student Affairs:
+          </span>
+          {OSA.map((document, index) => (
+            <li
+              key={index}
+              className="mb-2 ml-5"
+              style={{ fontWeight: 'bold' }}
```
Comment (miggyalino) [inline src/app/guidesite/page.tsx:204]: Same here

File: src/app/guidesite/page.tsx [hunk 8]
```diff
@@ -0,0 +1,363 @@
...
+        <ol className="list-decimal list-inside ml-6">
+          {facility.nestedInstructions.map((instruction, instructionIndex) => (
+            <li key={instructionIndex}>{instruction}</li>
+          ))}
+        </ol>
+      </li>
+    ))}
+  </ol>
+);
+const IcommpEquipment = () => {
```
Comment (miggyalino) [inline src/app/guidesite/page.tsx:112]: Can be done in a separate file

File: src/app/guidesite/page.tsx [hunk 9]
```diff
@@ -0,0 +1,363 @@
...
+        subtitle={'SAMAHAN'}
+        title={'GUIDESITE'}
+      />
+      <div className="text-blue font-sans px-8 md:px-32 py-10 bg-gray-100">
+        <div className="max-w-screen-lg mx-auto">
+          <section>
+            <h2 className="text-bold font-bold mb-4 pt-20">{title}</h2>
+
+            <ol className="list-decimal pl-5">
+              {steps.map((step) => (
```
Comment (miggyalino) [inline src/app/guidesite/page.tsx:249]: Mejo mahirap na basahin ang code dito kasi very nested na. Possible ba na maka gawa ng reusable component for steps then inside that component you can check kung ano ang pwede pa ma divide into reusable components

**Review by miggyalino — APPROVED** [review]
Approved but take note lang when doing components make sure to apply Single Responsibility Principle if possible para mas maintainable ang code. If you can notice, mejo deep na ang nesting ng mga elements and a lot is happening in one component, this can be improved pa by dividing it into more components para at least if something goes wrong, it's easy to spot the problem sa code. Pero nice work gihapon! @seanchvz

**Pull request merged**